package.json

```diff
@@ -1,79 +1,48 @@
 {
-  "name": "@morgan-stanley/desktopjs",
+  "name": "desktopjs",
   "title": "desktopJS",
-  "description": "Library for abstracting common container hosting",
-  "version": "2.5.0",
-  "publishConfig": {
-    "access": "public"
-  },
-  "main": "dist/desktop.js",
-  "types": "dist/desktopjs.d.ts",
+  "private": true,
   "license": "Apache-2.0",
   "author": "Morgan Stanley",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/Morgan-Stanley/desktopJS.git"
-  },
-  "bugs": {
-    "url": "https://github.com/Morgan-Stanley/desktopJS/issues"
-  },
-  "keywords": [
-    "container",
-    "desktop wrapper",
-    "desktop",
-    "openfin",
-    "hadouken",
-    "electron"
-  ],
-  "files": [
-    "dist"
-  ],
-  "dependencies": {
-    "@types/node": "*",
-    "@types/openfin": "*"
+  "scripts": {
+    "bootstrap": "lerna bootstrap",
+    "tslint": "lerna run tslint",
+    "clean": "lerna run clean",
+    "build": "lerna run build:lerna",
+    "test": "gulp test",
+    "start": "gulp server",
+    "docs": "gulp docs",
+    "watch": "lerna run --parallel watch"
   },
+  "dependencies": {},
   "devDependencies": {
-    "@types/jasmine": "^2.5.47",
     "codecov": "^3.0.0",
-    "dts-bundle": "^0.7.2",
+    "dts-bundle": "^0.7.3",
     "gulp": "^3.9.1",
     "gulp-clean": "^0.4.0",
-    "gulp-istanbul": "^1.1.1",
+    "gulp-istanbul": "^1.1.3",
     "gulp-istanbul-enforcer": "^1.0.3",
     "gulp-jasmine": "^4.0.0",
-    "gulp-rename": "^1.2.2",
+    "gulp-rename": "^1.4.0",
     "gulp-replace": "^1.0.0",
-    "gulp-sourcemaps": "^2.5.0",
-    "gulp-tslint": "^8.1.0",
+    "gulp-sourcemaps": "^2.6.4",
+    "gulp-tslint": "^8.1.3",
     "gulp-typedoc": "^2.0.2",
-    "gulp-typescript": "^4.0.1",
-    "gulp-uglify": "^3.0.0",
+    "gulp-typescript": "^5.0.0-alpha.3",
```

Comment:
Just noticing this.  It must have been tagged as latest.  If there are any objects I'll manually pin but greenkeeper will probably try and upgrade.

Reply:
yeah, it should be a non alpha at least

Reply:
Fixed